.github/workflows/release-events.yml

```
@@ -0,0 +1,109 @@
+
+jobs:
+  dispatch:
+    name: Post release event
+    # Skip noisy early-exit runs: auto-rollover that decided no rollover
+    # was needed would still fire this workflow otherwise. The tracking
+    # issue only cares about *actual* rollover commits. We keep
+    # test-signing failures always (that's the whole point of the sink).
+    if: >-
+      github.event_name == 'release'
+      || (github.event_name == 'workflow_run'
+          && github.event.workflow_run.conclusion != null
```

actionlint (1.7.12)

[error] got unexpected character '?' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)